src/components/TopBar/TopBarMenu.vue

```diff
@@ -377,7 +378,29 @@ export default {
 
 		showCallLayoutSwitch() {
 			return !this.$store.getters.isEmptyCallView
-		}
+		},
+
+	},
+
+	watch: {
+
+		'model.attributes.speaking'(speaking) {
+			// user is not speaking OR timeout is already running OR hand is not raised
+			if (!speaking || this.lowerHandTimeout !== null || !this.isHandRaised) {
+				return
+			}
+
+			this.lowerHandTimeout = window.setTimeout(() => {
+				this.lowerHandTimeout = null
+
+				if (!this.model.attributes.speaking || !this.isHandRaised) {
+					return
+				}
```

Comment:
I agree with `!this.model.attributes.speaking` point, and I think making it total count of 3 sec speaking will be better as in case of 3 sec of first speaking it can be triggered by any false alarm like background noise so having a threshold for 3 secs will be better